site/src/docs/patterns/navigation/building-navigation.mdx

```diff
@@ -0,0 +1,522 @@
+            href="/new-location"
+            active={active === '/new-location'}
+            id="sub-level-6"
+            label="Create a new location"
+            onClick={setActivePage}
+          />
+        </SideNavigation.MainLevel>
+        <SideNavigation.MainLevel
+          href="/settings"
+          active={active === '/settings'}
+          id="main-level-3"
+          label="Settings"
+          onClick={setActivePage}
+        />
+        <SideNavigation.MainLevel
+          external
+          href="https://tapahtumat.hel.fi/"
+          openInNewTab
+          openInNewTabAriaLabel="Opens in a new tab."
+          openInExternalDomainAriaLabel="Opens a different website."
+          id="main-level-4"
+          label="Tapahtumat.hel.fi"
+          openInExternalDomainAriaLabel="Opens a different website"
+          withDivider
+        />
+      </SideNavigation>
+    </div>
+  </div>);
+};
+
+<PlaygroundPreview>
+  <SideNavExample />
+</PlaygroundPreview>
+
+<ul style="margin-bottom:0;">
+  <li>
+    The learn more about the <strong>SideNavigation</strong> component, please refer to
+    <InternalLink href="/components/side-navigation">the component page.</InternalLink>
+  </li>
+</ul>
+
+### 5. Tabs (optional)
+
+Tabs effectively complement top-level navigation by categorizing related content at the same hierarchy level. Ensure tabs are clearly labeled, selectable, and highlight when active. Arrange them logically, typically left-to-right. Tabs should supplement, not replace, top-level navigation.
+
+<PlaygroundPreview>
+  <Tabs>
+    <Tabs.TabList style={{ marginBottom: 'var(--spacing-m)' }}>
+      <Tabs.Tab>Daycare</Tabs.Tab>
+      <Tabs.Tab>Pre-school</Tabs.Tab>
+      <Tabs.Tab>Basic education</Tabs.Tab>
+      <Tabs.Tab>Upper secondary</Tabs.Tab>
+      <Tabs.Tab>University</Tabs.Tab>
+    </Tabs.TabList>
+    <Tabs.TabPanel>
+      Daytime care for people who cannot be fully independent, such as children or elderly people.
+    </Tabs.TabPanel>
+    <Tabs.TabPanel>
+      A pre-school is an educational establishment offering early childhood education to children before they begin
+      compulsory education at primary school.
+    </Tabs.TabPanel>
+    <Tabs.TabPanel>
+      The objective of basic education in Finland is to support pupils&#39; growth towards humanity and ethically
+      responsible membership of society.
+    </Tabs.TabPanel>
+    <Tabs.TabPanel>
+      Upper secondary school studies last three to four years, preparing the students for the matriculation examination.
+    </Tabs.TabPanel>
+    <Tabs.TabPanel>
+      A high-level educational institution in which students study for degrees and academic research is done.
+    </Tabs.TabPanel>
+  </Tabs>
+</PlaygroundPreview>
+
+<ul style="margin-bottom:0;">
+  <li>
+    The learn more about the <strong>Tabs</strong> component, please refer to
+    <InternalLink href="/components/tabs">the component page.</InternalLink>
+  </li>
+</ul>
+
```

Comment:
Should I use tabs or side navigation?

Reply:
"When to use" examples would make it clearer, absolutely.